assessments/0006-gateway-api.md

```diff
@@ -0,0 +1,240 @@
+- Provide examples of great documentation as a reference
+- Identifies key improvements with the largest return on investment
+
+
+## How this document works
+
+The assessment is divided into three sections:
+
+- **Project documentation:** for end-users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+Each section rates content based on different [criteria](criteria.md).
+
+
+## Project documentation
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |  ✅  |     |     |     |
+| New user content           |     |     |  ✅  |     |     |
+| Content maintainability & site mechanics   |     |     |     |  ✅  |     |
+| Content creation processes |     |     |     |  ✅  |     |
+
+
+Scale:
+- 1 = (Is not present or requires significant work)
+- 3 = (Is present, but needs work)
+- 5 = (Is executed extremely well or no improvement required)
+
+
+### Comments
+
+**Information architecture**:
+
+The brief information is mentioned on [the Introductory](https://gateway-api.sigs.k8s.io/) page, but we need to organize it well. Also, [API conventions](https://gateway-api.sigs.k8s.io/concepts/guidelines/?h=api+conve#api-conventions) are mentioned in brief, great work!
+
+**New user content**: 
+
+We have a great, clearly labelled [Getting Started](https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/) page, which is awesome! However, the getting started guide is fairly high level and doesn't answer some of the following questions:
+- What are the prerequisites that need to be installed beforehand? Or What else do I need aside from Gateway Controller?
+* Where to go and what to do after reading the “Getting Started” guide?
+
+**Content maintainability**:
+
+Our documentation is searchable, which is great!
+
+**Content creation processes**:
+
+The [Contributing Guide](https://gateway-api.sigs.k8s.io/contributing/devguide/) contains really thorough documentation on contributing to both docs and the project itself! In the [README](https://github.com/kubernetes-sigs/gateway-api#readme), maintainers are not [clearly documented](https://github.com/kubernetes-sigs/gateway-api/blob/master/OWNERS_ALIASES) as well as where to find them.
+
+
+### Recommendations
+
+**Information architecture:**
+
+- The main task with information architecture is conceptualization and development as the documents are currently in different places. The following areas would establish a foundation:
+
+    –TODO: need to organize the subsections–
+
+    **Update: prepared a miro board [https://miro.com/app/board/uXjVO_1cS9k=/](https://miro.com/app/board/uXjVO_1cS9k=/)**
+
+    * Introduction
+    * Quick Start
+    * Concepts
+    * Tutorials
+    * Reference
+    * Contribute 
```

Comment:
Could we bring an image of the work you've done in miro in and remove the TODO comment?

It's OK to leave a link to the original board